try:
    a = [11, 23, 12, 12]
except Exception as e:
    print("sorry")
c = a[2]